GetIntoTeachingApiTests/Services/CandidateUpserterTests.cs

```diff
@@ -100,30 +124,24 @@ public void Upsert_WithPrivacyPolicy_SavesPrivacyPolicy()
             _upserter.Upsert(_candidate);
 
             policy.CandidateId = candidateId;
-            _mockCrm.Verify(mock => mock.Save(It.Is<CandidatePrivacyPolicy>(p => IsMatch(policy, p))), Times.Once);
-        }
 
-        [Fact]
-        public void Upsert_WithPhoneCallButMatchingQuotaIsAlreadyFull_DoesNotIncrementsCallbackBookingQuotaNumberOfBookings()
-        {
-            var candidateId = Guid.NewGuid();
-            var scheduledAt = DateTime.UtcNow.AddDays(3);
-            var phoneCall = new PhoneCall() { ScheduledAt = scheduledAt };
-            var quota = new CallbackBookingQuota() { StartAt = scheduledAt, NumberOfBookings = 5, Quota = 5 };
-            _candidate.PhoneCall = phoneCall;
-            _mockCrm.Setup(mock => mock.Save(It.IsAny<Candidate>())).Callback<BaseModel>(c => c.Id = candidateId);
-            _mockCrm.Setup(mock => mock.GetCallbackBookingQuota(scheduledAt)).Returns(quota);
-
-            _upserter.Upsert(_candidate);
-
-            _mockCrm.Verify(mock => mock.Save(It.IsAny<CallbackBookingQuota>()), Times.Never);
-            quota.NumberOfBookings.Should().Be(5);
+            _mockJobClient.Verify(x => x.Create(
+               It.Is<Job>(job => job.Type == typeof(UpsertModelJob<CandidatePrivacyPolicy>) && job.Method.Name == "Run" &&
+               IsMatch(policy, (string)job.Args[0])),
+               It.IsAny<EnqueuedState>()));
         }
 
-        private bool IsMatch(object objectA, object objectB)
+        private static bool IsMatch(object objectA, object objectB)
         {
             objectA.Should().BeEquivalentTo(objectB);
             return true;
         }
+
+        private static bool IsMatch<T>(T modelA, string modelBJson)

```

Comment:
These helpers are static in this test class but not in the others. Not sure if it is a big deal

Reply:
Yeah they could all be static, I'll change the other ones when I'm in there, or try and figure out a linter to sort this stuff 😅